ffig/FFIG.py

```diff
@@ -66,20 +66,45 @@ def get_template_output(class_name, template_name):
 
 # -- END Old code --
 
+
+def write_bindings_to_disk(api_classes, env, args, output_dir):
+    """ Input:
+    - api_classes
+    - environment to get templates from 
+    - args
+    - output_dir where to write to
+    write the bindings to disk, return Nothing """
+    for binding in args.bindings:
+        with open(os.path.join(output_dir, get_template_output(args.module_name, get_template_name(binding))), "w") as output_file:
+            template = env.get_template(binding)
+            s = render_api_and_obj_classes(api_classes, template)
+            output_file.write(s)
+
+def build_model_from_source(inp, module_name):
+    """Input:
+    - full path to source file
+    - module_name taken from args
+    Returns:
+    - model built from a clang.cindex TranslationUnit 
+        with a name, as defined in args
+    """
+    tu = clang.cindex.TranslationUnit.from_source(
+        inp, '-x c++ -std=c++14 -stdlib=libc++'.split())
+    model = cppmodel.Model(tu)
+    model.module_name = module_name
+
+    return model
 def main(args):
     cwd = os.getcwd()
     
     #FIXME: Remove the need for this constraint.
     if len(args.inputs) != 1:
         raise Exception("Multiple input files are currently not supported.")
 
-    #FIXME: Loop over files and extend the model once we can handle multiple input files.
+    # FIXME: Loop over files and extend the model once we can handle multiple
+    # input files.
     i = os.path.join(cwd, args.inputs[0])
```

Comment:
I'd usually reserve `i` for very local scopes and for integer counters. Can we call this `input_file`?